mvrx-rxjava2/build.gradle

```diff
@@ -0,0 +1,46 @@
+import com.airbnb.mvrx.JacocoReportTask
+
+apply plugin: "com.android.library"
+apply plugin: "kotlin-android"
+apply plugin: "kotlin-android-extensions"
+apply plugin: "jacoco"
+apply plugin: "com.vanniktech.maven.publish"
+
+android {
+    resourcePrefix "mvrx_"
+
+    testOptions {
```

Comment:
shouldn't this stuff be applied in the buildsrc plugin?

Reply:
@elihart This is all still in the main mvrx gradle file. These aren't set by the plugin. We could add it if we want.